tests/Amadeus/ClientTest.php

```diff
@@ -35,22 +35,24 @@ class ClientTest extends BaseTestCase
 {
     public function testCanCreateClient()
     {
-        $par = new Params([
```

Comment:
Everywhere in this library we use his construct of ([ 'the' => 'array' ]), so I don't want to have this changed in just this one file.